code/game/objects/effects/portals.dm

```diff
@@ -87,6 +88,8 @@
 
 	if(!M.simulated || iseffect(M))
 		. = FALSE
+	if(istype(M, /obj/effect)) // sparks should not go in portals
+		. = FALSE
```

Comment:
```suggestion
```

There'e `iseffect(M)` right above this

Reply:
Do you ever just feel really dumb after a code review